src/components/FolderInfo/index.tsx

```diff
@@ -1,6 +1,18 @@
 import "./FolderInfo.css";
 
-export const FolderInfo = ({ profile }) => {
+interface Profile {
+  owner?: {
+    name: string;
+    profileImageSource: string;
+  };
```

Comment:
`SharePage`에서 인라인으로 중첩해서 정의했던 `Owner`타입이 여기서도 쓰이고 있네요.
중복으로 인라인으로 타입을 작성하기보다 중복된 타입은 하나의 타입으로 만들어서 재사용 해주세요. 

```suggestion
type Owner = {
    profileImageSource: string;
    name: string;
};
  owner?: Owner
```